jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+
+==== Duplicate issues
+
+On Jira we used a plugin to detect potential duplicate issues
+On GitHub we will use link:https://github.com/actions-cool/issues-similarity-analysis[actions-cool/issues-similarity-analysis] to do the same thing as used on the link:https://github.com/jenkins-infra/helpdesk/blob/main/.github/workflows/issues-similarity.yml[Infra helpdesk].
+
+==== Dashboards
+ 
+A number of Jira dashboards are setup.
+GitHub projects (Beta) can be used to achieve a similar capability.
+
+- link:https://github.com/users/timja/projects/4[Core maintainers attention]
+- link:https://github.com/users/timja/projects/3[User experience project]
+
+Issues need to be added to a project this can be achieved with a script link:add-issue-to-project.sh[].
+
+The dashboards aren't as powerful as Jira and the sorting is limited. See link:https://github.com/github-community/community/discussions/8518[Can't currently sort issues by created / updated].
+
+==== Subscribing to select issues, e.g. label
+
+Users in Jira were able to configure notifications based off of a saved query, which was a very powerful feature.
+GitHub's built-in feature only allows you to subscribe to all issues or subscribe individually as you see an issue.
+
+There are a couple of workarounds:
+- link:https://github.com/orgs/github-community/discussions/16645[Subscribe to labels in GitHub]
+- link:https://github.com/bytecodealliance/subscribe-to-label-action[]
+- link:https://github.com/rseanhall/issue-label-watcher[]
+
+==== Project management in one view
+
+Previously Jira was used to track any bigger projects such as Guava upgrade and Java 11.
+This was complicated by some plugins using GitHub and some Jira to track issues.
+
+As of June 2022 515 repositories have GitHub issues, (21% of repositories). 46325 issues have been file in GitHub.
+
+GitHub projects can be used to group issues that are across components.
+If that project doesn't use GitHub issues then either:
+- A tracking issue can be created that links to a Jira epic or query
+- A repo is created for tracking individual issues and those issues link to an issue in the plugins issue tracker
+
+=== Missing matching capability from Jira
+
+==== Can't move issues without write access on both sides
+
+In Jira to move an issue to a more appropriate component then the component field was updated.
+
+In GitHub to move an issue you can either:
+1. Transfer it if you have write access on both repositories
+2. Refile the issue under a different user linking back to the old one.
+
+The Jira workflow relied on:
+1. Users updating the assignee so that default assignee logic was re-evaluated and the actual maintainer notified
+2. Maintainers manually configuring filters in Jira to notify them
+
+If we do wish to maintain the ability to move issues for anyone or at least org members then we would need to create a small app that has the ability to do this, deploy it somewhere, and configure a webhook at the organization level.
```

Comment:
That should be available to org members, or in a more restrictive manner, core maintainers (? at least these people have write permissions on named repositories above).

People reporting issues may not always know the appropriate repository to do that on, especially if an issue involves possibly lesser known core components for an end user, like stapler, winstone etc.

Reply:
I think no permission level required should be fine, that's what I did with the labels chat ops command.